OpenRA.Mods.Common/Widgets/WorldInteractionControllerWidget.cs

```diff
@@ -61,7 +61,16 @@ public override void Draw()
 			{
 				var a = worldRenderer.Viewport.WorldToViewPx(dragStart);
 				var b = worldRenderer.Viewport.WorldToViewPx(mousePos);
-				Game.Renderer.RgbaColorRenderer.DrawRect(a, b, 1, Color.White);
+
+				Color color;
+				if (modifiers.HasFlag(Modifiers.Alt) && !modifiers.HasFlag(Modifiers.Ctrl))
+					ChromeMetrics.TryGet("AltSelectionColor", out color);
```

Comment:
Querying this every render isn't great for performance - can we please cache these from the constructor?

Reply:
Done.